docs-samples/V4/JS/add-suggested-actions/app.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Botbuilder v4 SDK - Add suggested actions
+ * 
+ * This bot shows buttons the user can press to provide input.
```

Comment:
Add this: "These SuggestedActions buttons will disappear after the user clicked on any of the buttons. The text value of the button will be send as a message to the bot representing the user's choice. The bot can take action based this choice.